### ISSUE TITLE
where we go beyond bounties -- building a data-driven experiment machine

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#208]
# where we are now

right now gitcoin arguably has two main features: tips and bounties and two main actors: repo maintainers and developers.

# where we want to go

i view the _future_ of gitcoin as having many many tools that support open source software developers.  

(and many actors that particpate and benefit in ways that scale relative to their participation)

# how we get there

as a means of iterating towards building tools that developers _want_ to use that _provide value_, we are going to build things, we will dogfood them, and we will be relentlessly data driven and ego-less about which are working and which are not. 

in order to design this future, we need to figure out a way of evolving the community towards success.

- evolution had natural selection to select the best species to propogate forward.
- science has scientific method to select the best hypothesis' to propogate forward.
- gitcoin will have data to select the best tools to propogate forward.

here's a very hand-wavey view of how i see this selection mechanism being built into gitcoin

1. debate & identify opportunities
1. scope MVPs 
1. build/launch said MVPs.
1. measure relentlessly.
1. celebrate the tools that are working.  
1. sunset the tools that aren't working (but celebrate the learnings and the fun we had building them)

in parallel to each of the above, we will find a way to bake an incentive model into said process of entrepeneurship so that (1) the people involved in each experiment (2) the community at large benefit from their successes.

# what can we do in the near term to support this future?

in order to support all of this, we need to have KPIs (key performance indicators) built into each tool.  

as a means of prototyping the same, i have built the following into https://gitcoin.co/toolbox... which is live now:

![screen recording 2018-01-02 at 08 15 pm](https://user-images.githubusercontent.com/513929/34508154-c0a47d36-eff9-11e7-9d67-28f64fd09249.gif)

this prototype clearly leaves a lot to be desired.. but don't be fooled by the simplicity of the design.. there's data in them thur graphs!

onto the next evolution..

# design bounty

for each tool, what would a tool detail page look like that 
- shows off the tools' "brand"
- articulates the hypothesis behind the tool
- identifies the 'principal's (people) behind each tool
- identifies the metrics that track how the tool is performing.
  
on the index page, how do we surface tools that allow users to navigate the tools available on gitcoin?  
- do we use tags?
- do we have a 'trend'ing section?
- where do sunset'd tools live?

**Comment by gitcoinbot** [comment]
__This issue now has a funding of 0.15 ETH (128.63 USD) attached to it.__

 * If you would like to work on this issue you can claim it [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208).
 * If you've completed this issue and want to claim the bounty you can do so [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $12395.23 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by Elaniobro** [comment]
![](https://media.giphy.com/media/RgAl2ESJFnIQ0/giphy.gif)

**Comment by ayamane** [comment]
FYI, the toolbox link is not working.  Appears to be missing the ".co"

**Comment by owocki** [comment]
thanks @ayamane  -- updated

**Comment by gitcoinbot** [comment]
__The funding of 0.15 ETH (158.89 USD) attached has been shown interest by: [@owocki](https://gitcoin.co/profile/owocki).__ 

 If you are interested, please leave a comment to let the funder (@owocki) and the other parties involved why you're interested in working on this issue and your plans to resolve it.  If you don't leave a comment, the funder may not think you're too interested. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $40898.08 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by gitcoinbot** [comment]
__The funding of 0.15 ETH (158.89 USD) attached has been fulfilled by @owocki.__ 

 @owocki, please leave a comment to let the funder (@owocki) (and the other parties involved) that you've fulfilled the bounty.  If you don't leave a comment, the funder may expire your claim at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $40898.08 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by gitcoinbot** [comment]
__This claim for the funding of 0.15 ETH (158.89 USD) attached to this issue has been **rejected** and can now be claimed by someone else.__

 * If you would like to work on this issue you can claim it [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208).
 * If you've completed this issue and want to claim the bounty you can do so [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $40898.08 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by vs77bb** [comment]
@owocki should this one be closed out?

**Comment by owocki** [comment]
i dont think so.. i'd really like to see someone tackle this..

**Comment by thinkmassive** [comment]
What do you have in mind to fulfill the design bounty? Are you looking for a wireframe for each product detail page plus the index page? Copy & annotations?

**Comment by owocki** [comment]
@thinkmassive 

i'm looking for a wireframe + product detail page for *one* product.. lets start with 'bounties' perhaps

**Comment by vs77bb** [comment]
@owocki For reference, I've looked at this issue multiple times and just now realized you're looking to fund a *detail page* for existing tools. 

My sense was always that you were looking to fund design bounties for *new tool ideas*. 

Now, my guess is you'd like both to happen, ideally. May be worth breaking out into two separate bounties.

**Comment by gitcoinbot** [comment]
__Work has been started on the 0.15 ETH (88.43 USD) funding by__: 
 1. [@juilyoon](https://gitcoin.co/profile/juilyoon) 

 __Please work together__ and coordinate delivery of the issue scope. Gitcoin doesn't know enough about everyones skillsets / free time to say who should work on what, but we trust that the community is smart and well-intentioned enough to work together.  As a general rule; if you start work first, youll be at the top of the above list ^^, and should have 'dibs' as long as you follow through. 

 On the above list? Please leave a comment to let the funder (@owocki) and the other parties involved what you're working, with respect to this issue and your plans to resolve it.  If you don't leave a comment, the funder may expire your submission at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $6393.89 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by lazaridiscom** [comment]
I like your enthusiasm, and I've claimed this one (to have a balance for code-level work). Will give you some results in a few days, subjecting this "mini-startup-incubator".

I have a personal interest in the bounty-system, as I think that it can help me a lot, with different things. And I believe that the bounty-system, as is, needs some rework/reviesioning.

And thus, I will design (not much visually, but mostly functionally, spec/requirements wise) the product "gitcoin/bounty" (taking in account my experiences (positive and negative) so far. And give you overall feedback to what you've written here.

**Comment by lazaridiscom** [comment]
@owocki , this here is my result. It is possibly not what you have expected/requested. Instead of following the dictated process, I try here to "ring your bell" so that you adjust the overall process/sequence of reaching the goals.

And be aware that the fact "no one claims the bounty" can mean two things:

* The bounty is set to low (for the effort)
* The bounty affects a difficult or confusing issue

I will keep the overall structure of your document, and reply to some sections.

## "where are we now"

**gitcoin.co** a MVP for the bounties exists, which has several weaknesses and flaws (design, process, implementation)

(which i like, conceptually, and already use. The theoretical benefits the tool describe exist, despite hick-ups, barriers and some strange implementations/presentations)

## "where we want to go"

> i view the future of gitcoin as having many many tools that support open source software developers.
> (and many actors that particpate and benefit in ways that scale relative to their participation)

This sounds like a decent goal. Yet, other goals have priority (and should precede the above) :

* Stabilize the existent bounty-MVP
* Structure the overall ventures and ideas
* Clearly separate the toolbox from the bounties site
* Provide unified documentation & presentation 

additionally:

https://gitcoin.co/toolbox is already overloaded (too much information) and over-designed (more graphic design than information).

In order to avoid confusing the users which really just want to use the bounty-system, most of the things (including the issue here, which queries for new stuff) should be swapped out to an "umbrella" entity.

`gitcoin.co` should be **one** thing: the bounty-system, which belongs to an overall toolset, which should be named and hosted elsewhere. - call it for now: **ossbreed.org**).

`gitcoin.co` would then simply have a link back to `ossbreed.org`, thus it becomes clear that its a related project. Whoever is interested, reads then about more things/tools there.

## "how we get there"

Your description re evolution etc. and metrics and KPI is rational and vaild. What is invalid is "the point in time". It is not (yet) the time for this.

The suggested process is:

* Provide a clear structure for all this
* Clearly separate the tools, do not force stuff on users
* Simplify, strip-down, simplify, strip-down, again...

## "what can we do in the near term to support this future?"

* Give https://gitcoin.co/toolbox a home
* alternatively: make the information accessible immediately. A left-side navigation bar listing the tools, click, tool displays stuff&data (e.g. metrics)

### "for each tool, what would a tool detail page look like that"

This is an issue in itself, and should be placed on a dedicated resource, **after** providing the overall structure.

### "on the index page, how do we surface tools that allow users to navigate the tools available on gitcoin?"

* In no way! All tools which affect bounties, should work naturally, behind the scenes.
* the bounties landing page should be crystal-clear and efficient (and issue in itself

An one last thing/idea: those boxes of the tools in the toolbox should show immediately the relevant information (metrics) in form of numbers and graphs. Once can then click-in to get more data and descriptions.

This would mean: the toolbox mixes "gitcoin.co" related stuff (e.g. metrics), other stuff.tools (like new venture ideas etc.). Those 2 things should be separated.

-

(sidenote: My process: I read through the stuff 2 days ago. Worked with gitcoin the last 2 days. Today, right after waking up, I spended my "fresh brain" time with writing up all this.)

**Comment by gitcoinbot** [comment]
__Work for 0.15 ETH (130.63 USD) has been submitted by @lazaridiscom.__ 

 @lazaridiscom, please leave a comment to let the funder (@owocki) (and the other parties involved) that you've submitted you work.  If you don't leave a comment, the funder may expire your submission at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $9227.76 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by owocki** [comment]
@lazaridiscom thanks for the thoughtful note.  i dont think its really what i was looking for for the scoped bounty, but we always welcome feedback about the UX, brand architecture across multiple tools, and customer journey.  cc @pixelantdesign in case any of this is on her roadmap

**Comment by lazaridiscom** [comment]
@owocki , please do me a tiny favor: review what I wrote tomorrow again, and assess it in combination with the other issues, especially gitcoinco/gitcoinco/issues/26 (I used the quite hard word "unacceptable" intentionally).

See, i could have simply provided you what you asked for...

> i'm looking for a wireframe + product detail page for one product.. lets start with 'bounties' perhaps

... to get the bounty.

But this would not make any sense, as there are other things that need to be fixed first.

As for product ideas: they will pop up naturally, once gitcoin functions smoother and grows.

**Comment by gitcoinbot** [comment]
__The funding of 0.15 ETH (130.85 USD) attached to this issue has been approved & issued to @lazaridiscom.__  

 * Learn more at [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/208)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $7999.92 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by owocki** [comment]
accidently triggered the 'payout' github comment / email on my local. please ignore

**Comment by lazaridiscom** [comment]
all fine

**Comment by thelostone-mc** [comment]
@lazaridiscom 
> https://gitcoin.co/toolbox is already overloaded (too much information) and over-designed (more graphic design than information).

When you say too much information.. What do you mean? Cause in my head, I see you've got told their description and usage statistics. In my head like you suggested the statistics could be made more meaningful. But the content seems alright to me

And a bit more context on the graphic design being over loaded ?

The ossbreed umbrella (Rather than gitcoin being the umbrella itself)  and the other stuff makes sense to me. :)

**Comment by lazaridiscom** [comment]
I show you a counter-example. This here, has no info-overload/overdesign:

http://www.ti.com/product/LM73605/description

You can suck the information in, for hours, without any major distraction or indirection. If this site would be designed like gitcoin, you would not be able to read for long.

You will say now: "totally different domain". The please, look at this here:

https://bounties.network/

Clean and concise design, minimal information (ok, those folks need to add a little bit more). Actually a merger of gitcoin.co & bounties.network would bring out the ideal result.

Back to gitcoin.co as a standalone, concrete system for OSS bounties:

What is needed, to minimize the information overload is to split it up into 2 major parts (claimers and placers):

![image](https://user-images.githubusercontent.com/35095758/36133751-e53252c2-1088-11e8-9655-2e5c38441789.png)

"Claim Bounties" does not need the information for the "Put Bounties" (or "place bounties" ? ). And if any of the 2 parties want more, they go into community stuff. All this, must be directly relate to the bounties, except a link out to the "osstoolbox", for folks which has interest.

Having reduce the amount of "cards" in each 3 section (Claim Bounty, Put Bounty, Community), things will be already better.

Hope I could clarify a bit.

(i guess its "Place Bounties" and not "Put Bounties"... and "Community" could become "Project", so its clear that "gitcoin.co" is actually a contributable project, too)

**Comment by thelostone-mc** [comment]
http://www.ti.com/product/LM73605/description -> IMHO does need a lot of content as against gitcoin but yeah I get your point of not having too many stuff in the same basket ! Makes sense

Thanks for explaining that out ^_^

**Comment by lazaridiscom** [comment]
@vs77bb wrote far above: 

> My sense was always that you were looking to fund design bounties for new tool ideas.

yes, I thought the same when reading through this here. Now realized:

> i'm looking for a wireframe + product detail page for one product.. lets start with 'bounties' perhaps

That all this affects the **existent**  toolbox entries.

**Comment by lazaridiscom** [comment]
@owocki , you were unhappy with my 1st result... well. 2nd result follows:

I think I've found the way to solve several problems, and give you at the same time a wireframe/text for a new (and missing) tool:

[Gitcoin da Book.pdf](https://github.com/gitcoinco/web/files/1722202/Gitcoin.da.Book.pdf)

**Comment by lazaridiscom** [comment]
@owocki , I insist that this was an in-topic & good result! You should not be totally fixated on an bounty-description, especially when its a bit unclear. Results count. Read through this issue, and see the suggested new Tool: a book (which I think you should really start to write).

@thelostone-mc , @vs77bb , @PixelantDesign - I like to call-in a "trial". You 3 vote if my contributions within this issue (both, the off-topic and the in-topic) would justify the bounty.

(And as a sidenote: this should be workout someday, and official "mediator process". Because there will be the day where funder/claimer disagree...)

**Comment by thelostone-mc** [comment]
IMHO it sort of deviates from what @owocki had expected which was (quoting @vs77bb )
>For reference, I've looked at this issue multiple times and just now realized you're looking to fund a detail page for existing tools.
My sense was always that you were looking to fund design bounties for new tool ideas.
Now, my guess is you'd like both to happen, ideally. May be worth breaking out into two separate bounties.

You said that the entire idea has re-thought which IMO goes beyond this issue and doesn't essentially solve what @owocki  had asked for!
That said your idea of gitcoin.co being be one thing: the bounty-system and be a part of a bigger umbrella which encomposes tools makes sense to me.

Still not sure if this can be claimed under this bounty 
But In my eyes , I'd call it a success based on what you'd listed out 

PS: thanks for being patient and breaking down stuff for me :)

**Comment by lazaridiscom** [comment]
Hey, you've overseen the 2nd result: I actually suggested/-drafted a (new) Tool (based on "design bounty" section at the end of the original post) , which is missing: a book, for folks which prefers to read everything in one chunk)

> you were unhappy with my 1st result... well. 2nd result follows:
>
>I think I've found the way to solve several problems, and give you at the same time a wireframe/text for a new (and missing) tool:
>
> [Gitcoin da Book.pdf](https://github.com/gitcoinco/web/files/1722202/Gitcoin.da.Book.pdf)

P.S.: You're welcome. "Breaking down stuff" gives the ability to express oneself clearer (e.g. the next time)

**Comment by lazaridiscom** [comment]
Just for the case that you vote negative (don't dare!!!), I summarize what was asked for in context of the **existent** tools.

Because if you vote negative, then I still claim this bounty, by providing the result for the task below:

(@owocki wrote in the above discussion: )

# design bounty

for each tool, what would a tool detail page look like that

    shows off the tools' "brand"
    articulates the hypothesis behind the tool
    identifies the 'principal's (people) behind each tool
    identifies the metrics that track how the tool is performing.

on the index page, how do we surface tools that allow users to navigate the tools available on gitcoin?

    do we use tags?
    do we have a 'trend'ing section?
    where do sunset'd tools live?


**i'm looking for a wireframe + product detail page for one product.. lets start with 'bounties' perhaps**
